spec/index.html

```diff
@@ -574,7 +574,7 @@ <h2>Literals</h2>
         identifying a datatype that determines how the lexical form maps
         to a <a>literal value</a>, and</li>
       <li>if and only if the <a>datatype IRI</a> is
-        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code>, a
+        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code> (<code>rdf:langString</code>), a
```

Comment:
I suggest that we do not use the prefix form at this point.

Reply:
```suggestion
        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code>, a
```